source/objects/Model.cpp

```diff
@@ -94,7 +101,7 @@ Mesh::Mesh(
 								tri.tangents[j] = glm::normalize(glm::vec3(tri.e1[0], tri.e1[1], tri.e1[2]));
 							}
 
-							tri.uvs[j] = glm::vec2(verts[j]->texCoord.x, verts[j]->texCoord.y);
+							tri.uvs[j] = uvs[j];
```

Comment:
this shouldn't be needed now that it's passed to the constructor right?

Reply:
Yep, my mistake